src/components/PriceChart.tsx

```
@@ -37,472 +48,302 @@ const PriceChart: React.FC<PriceChartProps> = ({ assetName, timeFrame = '24h', o
+      case '30d': {
+        // Create approximately 6 evenly spaced ticks, plus start and end dates
+        const tickCount = 6;
+        const ticks: number[] = [];
+        for (let i = 0; i <= tickCount; i++) {
+          ticks.push(startMs + (i * (duration / tickCount)));
+        }
+        return ticks;
+      }
+      case 'all': // 90 days
+        // Create approximately 6 evenly spaced ticks
+        const tickCount = 6;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)